CMakeLists.txt

```diff
@@ -579,6 +579,14 @@ include(BuildCDT)
 libccdt_build()
```

Comment:
Something does not work in case of a big format:
```
tarantool> d:format(string.rep('a', 10000))
---
- !!binary YWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWFhYWHL
...
```

Reply:
This would be handled by suggestion of 1.5 passes from comment above.

Reply:
Added test for such case:

```diff
@@ -249,6 +249,16 @@ test:test("Formatting limits", function(test)
     len = ffi.C.tnt_datetime_strftime(T, buff, len + 1, fmt)
     test:is(len, 10, 'format(fmt) with non-NULL')
     test:is(ffi.string(buff), '01/01/1970', 'Epoch string (fmt)')
+
+    local bogus_fmt = string.rep('1', 10000)
+
+    len = ffi.C.tnt_datetime_strftime(T, nil, 0, bogus_fmt)
+    test:is(len, #bogus_fmt, 'format(fmt) with NULL and bogus format')
+
+    local s = T:format(bogus_fmt)
+    test:is(s, bogus_fmt, 'bogus format #1')
+    s = T:format(bogus_fmt..'%T')
+    test:is(s, bogus_fmt..'00:00:00', 'bogus format #2')
 end)
```